tests/x11regressions/groupwise/groupwise.pm

```diff
@@ -0,0 +1,36 @@
+# Groupwise tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Groupwise client check
+# Maintainer: Petr Cervinka <pcervinka@suse.com>
+
+use strict;
+use base "x11regressiontest";
+use testapi;
+use utils;
+
+sub run() {
+    select_console('root-console');
+    pkcon_quit;
+
+    # add repository and install groupwise
+    zypper_call("ar http://download.suse.de/ibs/SUSE:/Factory:/Head:/Internal/standard/ groupwise_repo");
```

Comment:
Well, there is https://en.opensuse.org/Package_repositories#Non-OSS

Also, interesting enough it seems groupwise has been part of SLE-11 and then dropped maybe?

So I understand we can't use this test for openSUSE, fine. How about testing it on different versions of SLE though? You only tested SLE12SP2 so far it seems. Please check at least SLE12SP3 then